generators/php/codegen/src/ast/Class.ts

```diff
@@ -20,6 +22,8 @@ export declare namespace Class {
         docs?: string;
         /* The class to inherit from if any */
         parentClassReference?: AstNode;
+        /* The traits that this class uses, if any */
+        usedTraits?: ClassReference[];
```

Comment:
nit: I would update this to just be `traits`, then the helper is just `addTrait`. We can assume it's used.